x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/rest/RestSqlTestCase.java

```diff
@@ -102,15 +104,16 @@ public void testNextPage() throws IOException {
                 + "\"mode\":\"" + mode + "\", "
             + "\"fetch_size\":2" + columnarParameter(columnar) + "}";
 
+        Number value = xContentDependentFloatingNumberValue(mode, 1f);
```

Comment:
Minor: maybe a more complicated float with integer and decimal parts?

Reply:
The value itslef (as float) comes from Elasticsearch itself, since ES computes the score. A wider range of values is covered here: https://github.com/elastic/elasticsearch/pull/48261/files#diff-1a813bc23d4c1d3facd693627b0d6ce4R124

Reply:
ah, sorry, completely missed that somehow...